templates/settings.py.erb

```diff
@@ -17,3 +17,4 @@ REDIS_HOST = "localhost"
 REDIS_PORT = "<%= scope['redis::port'] %>"
 REDIS_DB = <%= scope['pulpcore::redis_db'] %>
 WORKING_DIRECTORY = "<%= scope['pulpcore::cache_dir'] %>"
+REMOTE_USER_ENVIRON_NAME = 'HTTP_REMOTE_USER'
```

Comment:
This has `ENVIRON_NAME`, and your comment mentions `ENVIRONMENT_NAME`. Looks like `ENVIRON_NAME` is correct from the issue.